plutarch-test/plutarch-base/Plutarch/ListSpec.hs

```diff
@@ -1,70 +1,66 @@
 module Plutarch.ListSpec (spec, integerList) where
 
 import Test.Syd
+import Test.Syd.Hedgehog ()
 
 import Plutarch.List (pconvertLists, pfoldl')
 import Plutarch.Prelude
 import Plutarch.Test
 
+import Hedgehog (Property)
+
+import Data.List (find)
+
+import Plutarch.Test.Property.Gen (genList, integerGen)
+import Plutarch.Test.Property.Util (haskPlutEquiv, viaBothPartial, viaPEq)
+
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+
 integerList :: [Integer] -> Term s (PList PInteger)
 integerList xs = pconvertLists #$ pconstant @(PBuiltinList PInteger) xs
 
 spec :: Spec
 spec = do
   describe "list" $ do
-    let xs10 :: Term _ (PList PInteger)
-        xs10 = integerList [1 .. 10]
-    describe "type" . pgoldenSpec $ do
-      "phead" @| phead # xs10 @== pconstant @PInteger 1
-      "ptail" @| ptail # xs10 @== integerList [2 .. 10]
-      let matchP = pmatch (integerList [1, 3, 1]) $ \case
-            PSNil -> perror
-            PSCons x _ -> x
-      "pmatch" @| matchP @== pconstant @PInteger 1
-    describe "fun" . pgoldenSpec $ do
+    describe "properties" $ do
+      describe "find" $ do
+        it "plutarch level find mirrors haskell level find" findTest
+      describe "elemAt" $ do
+        it "plutarch level elemAt mirrors haskell level elemAt" elemAtTest
+    describe "goldens" . plutarchDevFlagDescribe . pgoldenSpec $ do
```

Comment:
The `describe "goldens"` should be removed.

Reply:
The `plutarchDevFlagDescribe` here is unfortunate (due to `pelemAt` tracing), but I guess we can't do anything about it until #260 is done to resolve #290.